JuliaStream.jl/src/CUDAStream.jl

```diff
@@ -0,0 +1,146 @@
+include("Stream.jl")
+using CUDA
+
+const CuData = StreamData{T,CuArray{T}} where {T}
+const TBSize = 1024::Int
+const DotBlocks = 256::Int
+
+function devices()
+  return !CUDA.functional(false) ? [] :
+         map(d -> "$(CUDA.name(d)) ($(repr(d)))", CUDA.devices())
+end
+
+function blocks(data::CuData{T})::Int where {T}
```

Comment:
What does this do and why do we need it? Again, we probably want to inline this so we don't introduce something that looks like a new API call. Might need a comment if it's an extra step we need to be aware of.

Reply:
It saves us from repeating `data.size ÷ TBSize` 6 times, it also needs the `÷` for integer division which is a pain to work with.
Given it's length, it does look to more readable when inlined.